server/server.go

```diff
@@ -158,8 +164,13 @@ func New(cfg Config) (*Server, error) {
 	}, []string{"protocol"})
 	prometheus.MustRegister(tcpConnections)
 
+	// add ipv4 support
+	if cfg.Network == "" {
+		cfg.Network = DefaultNetwork
+	}
```

Comment:
Can you add to `RegisterFlags()` with the default please, instead of this?

Reply:
yeah, sure